src/main/java/org/jboss/pnc/bifrost/common/scheduler/Subscriptions.java

```diff
@@ -88,7 +77,8 @@ public void submit(Runnable task) {
         backOffRunnable.setRunnable(internalTask);
         backOffRunnable.setCancelHook(() -> unsubscribe(subscription, UnsubscribeReason.NO_DATA_FROM_SOURCE));
 
-        ScheduledFuture<?> scheduledFuture = executor.scheduleAtFixedRate(backOffRunnable, 0, backOffRunnableConfig.getPollIntervalMillis(), TimeUnit.MILLISECONDS);
+        ScheduledFuture<?> scheduledFuture = executor.scheduleAtFixedRate(backOffRunnable, 0,
+                backOffRunnableConfig.getPollIntervalMillis(), TimeUnit.MILLISECONDS);
```

Comment:
When parameters are in multiple lines, there have to be one parameter per line.

Reply:
Proposed fix upstream: https://github.com/project-ncl/ide-config/pull/1